### ISSUE TITLE
Error cloning repository

### ISSUE THREAD

**Issue by sitz** [nkaz001/hftbacktest#5]
Hi there! Just spotted this repo recently and been trying to set it up and play around with few options of own.
But - seems like `git-lfs` used to track `*.pkl` files are causing some trouble in cloning the repo and throwing errors.

Would it be possible to update the references or another file option to make this repo easily pull-able please?
Appreciate the work on this framework!

Stacktrace:
```
(.env.3.9) ➜  hftbacktest git:(master) ✗ git reset --hard
Updating files: 100% (22/22), done.
Downloading example/usdm/bnbusdt_20221116.snapshot.pkl (19 KB)
Error downloading object: example/usdm/bnbusdt_20221116.snapshot.pkl (af67be7): Smudge error: Error downloading example/usdm/bnbusdt_20221116.snapshot.pkl (af67be7664c09d002afe64f8b2a956f87c0355571ae8a0bbcf818d2e9f3cab04): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

Errors logged to '/Code/hftbacktest/.git/lfs/logs/20230222T150946.286206.log'.
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: example/usdm/bnbusdt_20221116.snapshot.pkl: smudge filter lfs failed
```

**Comment by nkaz001** [comment]
I wanted to provide the example data to get started easily, but the file size is too big to upload to github. The files are uploaded to github-lfs, but unfortunately, now it's blocked because of the limit of github-lfs bandwidth. I'm thinking of just providing a few minutes' data as an example.

There is no problem cloning code just you don't have the example data files. For now, please see https://github.com/nkaz001/collect-binancefutures and collect some data by yourself to test it.

I will find a way to resolve this issue.

**Comment by sitz** [comment]
Yeah - trying to make those input files by processing data, but seems like usage of binary (`*.pkl`) + custom event data format for these makes it bit hard to manage another layer of inputs.

Is it possible to just feed a stream of full data feed across in standardized plain text (`*.csv`) format instead using Tardis data format and make use of them in framework to process of order requests accordingly instead?
1. Ticker / Orderbook to maintain latency and exchange activity
2. Public Trade files to simulate fills for outstanding limit orders

Downloadable data file formats example: https://docs.tardis.dev/downloadable-csv-files

**Comment by nkaz001** [comment]
Yup. But according to their document, quotes data is restructured from order book L2 feeds. It's not `bookTicker` stream data.

**Comment by adamlansky** [comment]
Hey @nkaz001 thanks a lot for sharing this beautiful work!
Trying to understand which of all the available book data types from Tardis is expected by the script - is that incremental_book_L2?

**Comment by nkaz001** [comment]
Right, that is `incremental_book_L2`. I made a tutorial about data. Please see https://github.com/nkaz001/hftbacktest/blob/master/examples/Data%20Preparation.ipynb By the way, I have no access to Tardis data now, please let me know if there is a problem to use that.

**Comment by nkaz001** [comment]
Delete LFS files.
